tests/python/pants_test/pantsd/pantsd_integration_test.py

```diff
@@ -355,7 +355,7 @@ def test_pantsd_invalidation_file_tracking(self):
             ctx.checker.assert_started()
 
             # See comment in `test_pantsd_invalidation_pants_toml_file`.
-            time.sleep(15)
+            time.sleep(30)
```

Comment:
I made a recent attempt at bumping these, without luck.

It might be a good idea to assign me a ticket to more deeply investigate these, and then skip them for now.